src/track/text-track.js

```diff
@@ -46,6 +46,8 @@ export default class TextTrack extends Track {
    */
   constructor(settings: Object = {}) {
     super(settings);
+    // some of the OTT dash manifests do not return a label, we will take the language instead.
```

Comment:
don't think the comment should be here, not relevant in the OTT/OVP context.
You can just say: 

> use language tag if no display label is available

This is correct to any use case